authors/tests/test_login.py

```diff
@@ -0,0 +1,13 @@
+from django.test import TestCase, Client
+
+class TestLogin(TestCase):
+    def setUp(self):
+        self.client = Client(content_type="application/json")
+        self.user = {
+            "email": "jake@jake.jake",
+            "password": "jakejake"
+        }
+    
+    def test_login_user_returns_201_status_code(self):
```

Comment:
`201 `is supposed to be used when ;
>The request has been fulfilled and has resulted in one or more new resources being created . You can read more [here](https://httpstatuses.com/201)

Since loging in doesnot create a new resource, a `200` would make more sense